tests/Public/New-GuestConfigurationPolicy.Tests.ps1

```diff
@@ -696,13 +702,13 @@ Describe 'New-GuestConfigurationPolicy' {
                         $fileName = $ContentUri.Split('/')[-1]
                         $filePath = Join-Path -Path $defaultDefinitionsPath -ChildPath $fileName
                         Invoke-WebRequest $ContentUri -OutFile $filePath
+                    }
 
+                    It 'Should include contentManagedIdentity in the result object and exclude Arc machines' {
                         $basePolicyParameters['ManagedIdentityResourceId'] = 'myManagedIdentity'
```

Comment:
The way the base parameters are getting used here is actually wrong.
The base parameters should not be updated like this within the tests.
The base parameters are meant to stay the same and the tests can clone them as needed.
I think you've had to add a lot of extra code here because someone updated the base parameters.
Instead of adding the extra code to remove the parameters after they have been erroneously added, can we please fix the root cause that the base parameters should not be used as the test parameters here?
You can see in all the previous Contexts and test cases before this one how the baseParameters are copied and then added to in a new objects, but they shouldn't be getting changed like this